README.md

```diff
@@ -10,7 +10,13 @@
 
 ### List of Available Components
 
-View available React Components [here](http://react.carbondesignsystem.com). Usage information is available when you click the blue **?** icon in the top right corner of the selected component.
+View available React Components [here](http://react.carbondesignsystem.com). You can see usage information in several ways:
+
+1. Clicking the blue **Show Info** icon in the top right corner of the selected component. You can see the list of available React props
+2. Clicking **STORY** tab at the bottom. You can see some example code there
+3. Clicking **KNOBS** tab at the bottom and changing values there. Most knobs are shown as something like `Button kind (kind)`, where `kind` is the name of React prop
```

Comment:
Maybe include `the` before the `**____** tab` lines just to help with how the sentence flows?